modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/services/AccountsService.java

```diff
@@ -246,16 +246,6 @@ public Account save(UUID accId, ChangeUserRequest changes) {
         try (Transaction tx = txMgr.txStart()) {
             Account acc = accountsRepo.getById(accId);
 
-            String oldEmail = acc.getEmail();
-            String newEmail = changes.getEmail();
-
-            if (!oldEmail.equals(newEmail)) {
-                Account accByEmail = accountsRepo.getByEmail(oldEmail);
```

Comment:
Why this code was removed? How we can protect that user "A" will not set email as user "B"?

Reply:
All checks will be done by index on save